packages/ui5-application-inquirer/src/prompts/prompts.ts

```diff
@@ -344,7 +348,7 @@ function getTargetFolderPrompt(targetDir: string, validateFioriAppFolder?: boole
         default: (answers: UI5ApplicationAnswers) => answers.targetFolder || targetDir,
         validate: async (target, { name = '' }: UI5ApplicationAnswers): Promise<boolean | string> => {
             if (name.length > 2) {
-                return await validateTargetFolder(target, name, validateFioriAppFolder);
+                return await validateTargetFolderForFioriApp(target, name, validateFioriAppFolder);
```

Comment:
Why has the name changed? Fiori App is implied so the name change is not required. The other functions names have not been updated similarly?